src/GitHub.App/Models/Account.cs

```diff
@@ -75,11 +85,22 @@ public void CopyFrom(IAccount other)
         {
             if (!Equals(other))
                 throw new ArgumentException("Instance to copy from doesn't match this instance. this:(" + this + ") other:(" + other + ")", nameof(other));
+            
             OwnedPrivateRepos = other.OwnedPrivateRepos;
             PrivateReposInPlan = other.PrivateReposInPlan;
             IsOnFreePlan = other.IsOnFreePlan;
             HasMaximumPrivateRepositories = other.HasMaximumPrivateRepositories;
             Avatar = other.Avatar;
+
+            var otherAccount = other as Account;
+            if (otherAccount != null)
+            {
+                _bitmapSourceSubscription.Dispose();
+
+                _bitmapSourceSubscription = otherAccount.BitmapSource
+                    .ObserveOn(RxApp.MainThreadScheduler)
+                    .Subscribe(x => Avatar = x);
+            }
```

Comment:
If we are copying this `Account` from another `Account` instance. We should cancel the subscription to the previous `BitmapSource` and start a new subscription against the new `BitmapSource`.